tests/framework/clients/bucket.go

```diff
@@ -146,3 +148,35 @@ func (b *BucketOperation) CheckOBMaxObject(obcName, maxobject string) bool {
 	fetchMaxObject, _ := b.k8sh.GetResource("ob", obName, "--output", "jsonpath={.spec.endpoint.additionalConfig.maxObjects}")
 	return maxobject == fetchMaxObject
 }
+
+// Checks the bucket notifications for an bucket
+func (b *BucketOperation) CheckBucketNotifications(namespace, storeName, obcName, bucketname, notificationName, s3endpoint string) bool {
```

Comment:
should probably move all notification provisioning operations here.
will do that as part of the refactoring PR: https://github.com/rook/rook/pull/9150

Reply:
okay

Reply:
we should do that in later work. will add as priority 2 to: https://gist.github.com/yuvalif/c4c7aed3dbc956a65399865a77c6a929